parts/dcosmastervars.t

```diff
@@ -36,8 +46,15 @@
     "masterNSGName": "[concat(variables('orchestratorName'), '-master-nsg-', variables('nameSuffix'))]",
     "masterPublicIPAddressName": "[concat(variables('orchestratorName'), '-master-ip-', variables('masterEndpointDNSNamePrefix'), '-', variables('nameSuffix'))]",
     "apiVersionStorage": "2015-06-15",
-    "storageAccountBaseName": "[uniqueString(concat(variables('masterEndpointDNSNamePrefix'),variables('location'),variables('orchestratorName')))]",
-    "masterStorageAccountExhibitorName": "[concat(variables('storageAccountBaseName'), 'exhb0')]",
+{{if .HasWindows}}
```

Comment:
These additional windows-specific vars should go inside the existing `{{if .HasWindows}}` block above. (see line 26)